lib/fs.ts

```diff
@@ -69,6 +68,6 @@ export async function walk(dir: string): Promise<Array<string>> {
     .then(processFiles)
     .then(flattenAndRemoveSymlinks)
     .catch(err => {
-      throwError(err);
+      throw err;
```

Comment:
TODO: Maybe this should be a `FileSystemError`

Reply:
Yeah that sound reasonable to me 👍 I think that's a safe change